src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  static died() {
+    Animal.alive = Animal.alive.filter(({ health }) => health > 0);
+  }
+
+  health = 100;
+
+  constructor(name) {
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hidden = false;
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(bittenAnimal) {
+    if (bittenAnimal instanceof Carnivore || bittenAnimal.hidden === true) {
```

Comment:
```suggestion
    if (bittenAnimal instanceof Carnivore || bittenAnimal.hidden) {
```
You don't need to compare with true as it's already boolean